Odata-docs/webapi-8/tutorials/basic-crud.md

```diff
@@ -398,14 +392,8 @@ Note that for the built-in routing conventions to route the request successfully
 [HttpGet("odata/Customers({id})")]
 public ActionResult Get([FromRoute] int id)
 {
-    var customer = db.Customers.SingleOrDefault(d => d.Id == id);
-
-    if (customer == null)
-    {
-        return NotFound();
-    }
-
-    return Ok(customer);
+    var item = customers.AsQueryable().Where(c => c.Id == id);
+    return SingleResult.Create(item);
```

Comment:
There's no reason for this change. `$expand` works okay even when the return type is `ActionResult`.